roles/seqreports/tasks/2_install_config.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+- name: ensure seqreports config dir exists
+  file:
+    state: directory
+    path: "{{ seqreports_config_root }}"
+
+- name: deploy seqreports app config
+  template:
+    src: "templates/app.config.j2"
+    dest: "{{ seqreports_config_root }}/app.config"
+
+- name: deploy seqreports logger config
+  template:
+    src: "templates/logger.config.j2"
+    dest: "{{ seqreports_config_root }}/logger.config"
+
+- name: ensure seqreports log dir exists
+  file:
+    path: "{{ seqreports_log_dir }}"
+    state: directory
+
+- name: set log file path
```

Comment:
you probably don't want to it this way? instead, use the Jinja templating functionality for `logger.config` (like it's done for `app.config`)

Reply:
Hm, ah, yeah. Perhaps I can write a ticket about this as well?

Reply:
I guess you could.. although I think it's less work to actually fix it 😉

Reply:
Actually, I prefer to write tickets. Taking all the comments in consideration would be a substantial work.